tests/multi_bluetooth/stress_log_filesystem.py

```diff
@@ -0,0 +1,185 @@
+# Test BLE GAP connect/disconnect
+
+from micropython import const
+import time, machine, bluetooth, os
+
+TIMEOUT_MS = 10000
+
+LOG_PATH_INSTANCE0 = "stress_log_filesystem_0.log"
+LOG_PATH_INSTANCE1 = "stress_log_filesystem_1.log"
+
+_IRQ_CENTRAL_CONNECT = const(1)
+_IRQ_CENTRAL_DISCONNECT = const(2)
+_IRQ_GATTS_WRITE = const(3)
+_IRQ_GATTS_READ_REQUEST = const(4)
+_IRQ_PERIPHERAL_CONNECT = const(7)
+_IRQ_PERIPHERAL_DISCONNECT = const(8)
+_IRQ_GATTC_SERVICE_RESULT = const(9)
+_IRQ_GATTC_SERVICE_DONE = const(10)
+_IRQ_GATTC_CHARACTERISTIC_RESULT = const(11)
+_IRQ_GATTC_CHARACTERISTIC_DONE = const(12)
+_IRQ_GATTC_READ_RESULT = const(15)
+_IRQ_GATTC_READ_DONE = const(16)
+_IRQ_GATTC_WRITE_DONE = const(17)
+
+SERVICE_UUID = bluetooth.UUID("A5A5A5A5-FFFF-9999-1111-5A5A5A5A5A5A")
+CHAR_UUID = bluetooth.UUID("00000000-1111-2222-3333-444444444444")
+CHAR = (
+    CHAR_UUID,
+    bluetooth.FLAG_READ | bluetooth.FLAG_WRITE | bluetooth.FLAG_NOTIFY | bluetooth.FLAG_INDICATE,
+)
+SERVICE = (
+    SERVICE_UUID,
+    (CHAR,),
+)
+SERVICES = (SERVICE,)
+
+
+waiting_events = {}
+log_file = None
+
+def write_log(*args):
+    if log_file:
+        print(*args, file=log_file)
+        log_file.flush()
+
+last_file_write = 0
+
+def periodic_log_write():
+    global last_file_write
+    t = time.ticks_ms()
+    if time.ticks_diff(t, last_file_write) > 50:
+        write_log('tick')
+        last_file_write = t
+
+
+def irq(event, data):
+    write_log('event', event)
+
+    if event == _IRQ_CENTRAL_CONNECT:
+        print("_IRQ_CENTRAL_CONNECT")
+        waiting_events[event] = data[0]
+    elif event == _IRQ_CENTRAL_DISCONNECT:
+        print("_IRQ_CENTRAL_DISCONNECT")
+    elif event == _IRQ_PERIPHERAL_CONNECT:
+        print("_IRQ_PERIPHERAL_CONNECT")
+        waiting_events[event] = data[0]
+    elif event == _IRQ_PERIPHERAL_DISCONNECT:
+        print("_IRQ_PERIPHERAL_DISCONNECT")
+    elif event == _IRQ_GATTC_SERVICE_RESULT:
+        # conn_handle, start_handle, end_handle, uuid = data
+        if data[-1] == SERVICE_UUID:
+            print("_IRQ_GATTC_SERVICE_RESULT", data[3])
+            waiting_events[event] = (data[1], data[2],)
```

Comment:
please remove trailing comma on RHS, and then run `black -l99` on this file (to fix other formatting issues)